app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -76,6 +76,10 @@
     private ArrayList<String> mAssistantRequests = new ArrayList<>();
     private ArrayAdapter<String> mAssistantRequestsAdapter;
 
+    private String mDeviceInstanceId = "your device id";
+    private String mDeviceModelId = "your device model id";
+    private String mLanguageCode = "en-US";
```

Comment:
private final static String LANGUAGE_CODE = "en-US";

Reply:
I think I have some of the same changes in a recent push. Maybe this commit should be rebased.

Reply:
@Fleker OK. Should I close this PR ?

Reply:
@mopemope can you try to rebase?

Reply:
Yes. I can do it.
Should I rebase after waiting for @Fleker commit ? or should I rebase my commit ?

Reply:
As it turns out, I never made this change, so this looks fine. Although I do agree with Proppy in making it a static final variable.